File: app/src/main/java/org/fossasia/susi/ai/skills/feedback/adapters/viewholders/AllReviewsViewHolder.java
package org.fossasia.susi.ai.skills.feedback.adapters.viewholders;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.LinearLayout;
import android.widget.TextView;

import org.fossasia.susi.ai.R;

import butterknife.BindView;
import butterknife.ButterKnife;

/**
 * Created by arundhati24 on 27/06/2018
 */

public class AllReviewsViewHolder extends RecyclerView.ViewHolder {

    @BindView(R.id.itemFeedback)
    public LinearLayout itemFeedback;

    @BindView(R.id.tvInitials)
    public TextView initials;

    @BindView(R.id.tvEmail)
    public TextView feedbackEmail;

    @BindView(R.id.tvDate)
    public TextView feedbackDate;

    @BindView(R.id.tvFeedback)
    public TextView feedback;

    public AllReviewsViewHolder(View itemView) {
        super(itemView);
        ButterKnife.bind(this, itemView);
    }
}

File: app/src/main/java/org/fossasia/susi/ai/skills/feedback/adapters/viewholders/AllReviewsViewHolder.kt
package org.fossasia.susi.ai.skills.feedback.adapters.viewholders

import android.view.View
import android.widget.ImageView
import android.widget.LinearLayout
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import kotterknife.bindView
import org.fossasia.susi.ai.R

class AllReviewsViewHolder(itemView: View) : androidx.recyclerview.widget.RecyclerView.ViewHolder(itemView) {

    val itemFeedback: LinearLayout by bindView(R.id.itemFeedback)
    val avatar: ImageView by bindView(R.id.avatar)
    val feedbackEmail: TextView by bindView(R.id.tvEmail)
    val feedbackDate: TextView by bindView(R.id.tvDate)
    val feedback: TextView by bindView(R.id.tvFeedback)
}
